Framework/jmaqs-selenium/src/test/java/com/magenic/jmaqs/selenium/SeleniumConfigTest.java

```diff
@@ -41,40 +40,87 @@ public void getBrowserName() {
   /**
    * Web site base.
    */
-  @Test
+  @Test(groups = TestCategories.Selenium)
   public void getWebsiteBase() {
 
     String website = SeleniumConfig.getWebSiteBase();
 
     Assert.assertTrue(website.equalsIgnoreCase("http://magenicautomation.azurewebsites.net/"));
   }
 
+  /**
+   * Hub Url.
+   */
+  @Test(groups = TestCategories.Selenium)
+  public void getHubUrl() {
+
+    String hubUrl = SeleniumConfig.getHubUrl();
+
+    Assert.assertTrue(hubUrl.equalsIgnoreCase("http://ondemand.saucelabs.com:80/wd/hub"));
+  }
+
+  /**
+   * Command timeout.
+   */
+  @Test(groups = TestCategories.Selenium)
+  public void getCommandTimeout() {
+
+    int timeout = SeleniumConfig.getCommandTimeout();
+
+    Assert.assertNotNull(timeout);
+  }
+
   /**
    * Driver hint path.
-   *///FIXME: Commenting out test until repaired.  Expected result does not make sense.
-  /*@Test
+   */
+  @Test(groups = TestCategories.Selenium)
   public void getDriverHintPath() {
```

Comment:
This is a bad implementation for a unit test.  Note the fix me in the javadoc area.  It looks for the resources folder in jmaqs-selenium's module root `./jmaqs-selenium/Resources` instead of `./jmaqs-selenium/src/test/resources`.  Please correct.